src/components/input/Input.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+
+export default function Input(input) {
```

Comment:
better pass a whole input instead each input parameter separately. (make ```Input({input})``` instead of ```Input(input)```)